frontend/assets/javascripts/src/modules/analytics/ga.es6

```diff
@@ -0,0 +1,68 @@
+import * as user  from 'src/utils/user'
+import * as cookie from 'src/utils/cookie'
+
+const dimensions = {
+    signedIn: 'dimension1',
+    signedOut: 'dimension2',
+    ophanPageViewId: 'dimension3',
+    ophanBrowserId: 'dimension4',
+    platform: 'dimension5',
+    identityId: 'dimension6',
+    isLoggedOn: 'dimension7'
+};
+export function init() {
+    let guardian = window.guardian;
+
+    /*eslint-disable */
+    (function (i, s, o, g, r, a, m) {
+        i['GoogleAnalyticsObject'] = r;
+        i[r] = i[r] || function () {
+                (i[r].q = i[r].q || []).push(arguments)
+            }, i[r].l = 1 * new Date();
+        a = s.createElement(o),
+            m = s.getElementsByTagName(o)[0];
+        a.async = 1;
+        a.src = g;
+        m.parentNode.insertBefore(a, m)
+    })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
+    /*eslint-enable */
+
+    ga('create', guardian.googleAnalytics.trackingId, {
```

Comment:
Can we give the tracker, even though there is only one, a name.
Just to be explicit in what we call.